services/distribution/src/main/java/app/coronawarn/server/services/distribution/statistics/file/MockStatisticJsonFileLoader.java

```diff
@@ -40,7 +40,7 @@ public JsonFile getFile(StatisticType statisticType) {
     try {
       return new JsonFile(resource.getInputStream(), "local");
     } catch (IOException e) {
-      throw new RuntimeException(String.format("Failed to load Local JSON from path %s",
+      throw new JsonFileLoaderException(String.format("Failed to load Local JSON from path %s",
           serviceConfig.getStatistics().getStatisticPath()), e);
```

Comment:
move the message into the custom exception
```suggestion
      throw new JsonFileLoaderException(serviceConfig.getStatistics().getStatisticPath(), e);
```